.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -76,6 +76,11 @@ on:
         default: '{}'
         required: False
 
+      run_condition:
```

Comment:
"skip-run:"

Reply:
Fixed in 01c5a4df09765afb0069410e2feec8fe257effc6